testsuite/integration-tests/src/test/java/org/jboss/resteasy/test/providers/jaxb/JaxbCacheTest.java

```diff
@@ -39,9 +40,13 @@
         WebArchive war = TestUtil.prepareArchive(JaxbCacheTest.class.getSimpleName());
         war.addClass(JaxbCacheTest.class);
         // Arquillian in the deployment
-        war.addAsManifestResource(PermissionUtil.createPermissionsXmlAsset(new ReflectPermission("suppressAccessChecks"),
+        war.addAsManifestResource(PermissionUtil.createPermissionsXmlAsset(
```

Comment:
With PR https://github.com/resteasy/Resteasy/pull/1126 no additional permissions are needed for this test.